docs/tour/why_wasmcloud.mdx

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 3
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Hello Universe
+
+You've now built a simple microservice with wasmCloud that has common capabilities (handle HTTP requests, interface with a Key-Value store) that many cloud native applications need. So, why wasmCloud? What makes this application that you built different from similar apps on other platforms? Beyond providing an easy-to-use platform that's quick to get started with, we also want to make sure that you understand that the biggest benefits of wasmCloud come after you've built your application.
+
+Consider, now that you have your little microservice, what you may have to do to actually deploy that to production in the cloud. If you're building this application in a modern language and packaging it up into a container, you may have the following hurdles to consider _before you even get to the point of writing your features in the first place_:
+
+1. What CPU architecture am I going to be deploying to? Do I need to build for x86, ARM, or both?
+2. What operating system am I going to be deploying to? Do I need to support more than just Linux?
+3. What cloud am I going to deploy to? What if I need to deploy to multiple clouds?
+4. Do I need to be able to vertically scale my application?
+5. Do I need to be able to horizontally scale across multiple instances of my application? What about high availability? How will I handle communication between instances of my application?
+
+**All of the above questions should not influence your application's functional logic, don't need to be considered until after you've already built your application with wasmCloud, and can be freely changed at runtime when you need to.**
+
+### CPU Architecture and Operating System Support
+
+wasmCloud applications compile to [WebAssembly](https://webassembly.org/), which is a completely platform agnostic binary format. The wasmCloud runtime itself releases binaries for Linux, Mac, Windows, x86_64, and aarch64 machines.
+
+### Cloud Support
+
+wasmCloud applications can be deployed to any cloud, edge, bare-metal, or on-prem machine that can run the wasmCloud runtime. This means that you can deploy your application to any cloud provider, or even to your own datacenter, without having to worry about the underlying infrastructure.
+
+### Vertical Scaling
+
+Each wasmCloud actor is a self-contained unit of execution. This means that you can scale your application vertically by simply running more instances of your actor. The wasmCloud runtime will automatically load balance requests across all instances of your actor.
```

Comment:
I'm not sure I would call this vertical scaling 🤔  I get what you're saying: increasing the number of actors on a single host increases its ability to handle concurrent requests (up to a point), analogous to slotting in more RAM in a baremetal server. I'm used to vertical scaling referring specifically to increasing hardware capabilities, which isn't happening here. Increasing actor instance count feels more analogous to increasing the number of servers/nodes in a cluster. Thoughts?

Reply:
When you put it that way I think the analogy breaks down, perhaps I can boil this down to the difference between distributing an application and scaling an application instead of using the more infrastructure-focused words